### PR TITLE
Add config for payments feature toggle

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-813

### Change description ###
Just adding the config in this pr. 
In the next pr, use the config to process payments.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -180,3 +180,7 @@ variable "appinsights_instrumentation_key" {
   description = "Instrumentation key of the App Insights instance this webapp should use. Module will create own App Insights resource if this is not provided"
   default     = ""
 }
+
+variable "process_payments_enabled" {
+  default = "true"
```
Comment (nitinprabhu) [inline infrastructure/variables.tf]: may be disable it for now and turn it on when needed ?

Comment (aliveni) [reply infrastructure/variables.tf]: you mean disable by default and enable on aat and demo?
we have to enable on aat and demo for QA

Comment (doncem) [reply infrastructure/variables.tf]: There's always edgy case whether one should default to `true` or `false`

**Comment by aliveni** [comment]
> You would need to update helm config too. Can be done later as the app is not using it yet :)

@nitinprabhu updated helm config

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -133,6 +134,7 @@ containers:
       jurisdiction: BULKSCAN
       poBox: BULKSCANPO
       ocrValidationUrl: ${OCR_VALIDATION_URL_BULKSCAN_SAMPLE_APP}
+      payments-enabled: true
```
Comment (doncem) [inline src/main/resources/application.yaml]: perhaps best to stick to common naming convention at least within same config scope even though i **do** prefer hyphens

**Comment by aliveni** [comment]
> Where's the chart bump gone?

it's overridden by the other pr (merged to master) 😞 Will update shortly

**Pull request merged**